Task8/src/main/java/org/andersen/dao/impl/UserDaoImpl.java

```diff
@@ -0,0 +1,54 @@
+package org.andersen.dao.impl;
+
+import org.andersen.connection.ConnectionFactory;
+import org.andersen.dao.UserDao;
+import org.andersen.model.User;
+
+import java.sql.*;
+
+public class UserDaoImpl implements UserDao {
+    private Connection connection = ConnectionFactory.getConnection();
+    private static final String SQL_INSERT_USER =
+            "INSERT INTO public.\"User\"(id, name, creation_date) VALUES (DEFAULT, '%s', '%s')";
```

Comment:
What is default value here?

Reply:
I used it since I had to insert auto-incremented value for SERIAL type of ID field, but I've done inserting with a whole User object, so DEFAULT keyword isn't needed anymore.